library/seq.lisp

```diff
@@ -253,6 +253,14 @@ a new `Seq` instance."
                             (iter:zip! (iter:into-iter a)
                                        (iter:into-iter b)))))))
 
+  (define-instance (Into (Seq :a) (List :a))
+    (define (into lst)
```

Comment:
One small nit- the parameter should probably be named `seq` or similar since the input isn't a list, or a vector in the following instance.